docs/guides/createawiretap.md

```diff
@@ -0,0 +1,99 @@
+# Creating a Wiretap
+In this guide we will create a Wiretap to monitor the Cloudevent traffic happening within our bridge.
+
+
+## What is a Wiretap?
+A Wiretap is a powerful debugging tool/methodology that can be used to understand the flow of events through the system by subscribing to all of the events that pass through the associated `Broker`. This is accomplished by the use of a `Trigger` to route all of the events into a logging service.
+
+
+Consider the following illustration:
```

Comment:
Can we update this to use the same style as the other guides? I think Seb mentioned using Excalidraw.

Reply:
done :)